CSharp/DevVmPowershell/DevVmPsModules/Cmdlets/ShowInstallerAndInstanceRelativityVersionAreEqualModule.cs

```diff
@@ -0,0 +1,114 @@
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 1,
+			HelpMessage = "Username of the Relativity Admin")]
+		public string RelativityAdminUserName { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 2,
+			HelpMessage = "Password of the Relativity Admin")]
+		public string RelativityAdminPassword { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 3,
+			HelpMessage = "Username of the Relativity Sql Account")]
+		public string SqlAdminUserName { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 4,
+			HelpMessage = "Password of the Relativity Sql Account")]
+		public string SqlAdminPassword { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 5,
+			HelpMessage = "Relativity Version of the Installer File")]
+		public string InstallerRelativityVersion { get; set; }
+
+		protected override void ProcessRecordCode()
+		{
+			//Validate Input arguments
+			ValidateInputArguments();
+
+			IConnectionHelper connectionHelper = new ConnectionHelper(
+				relativityInstanceName: RelativityInstanceName,
+				relativityAdminUserName: RelativityAdminUserName,
+				relativityAdminPassword: RelativityAdminPassword,
+				sqlAdminUserName: SqlAdminUserName,
+				sqlAdminPassword: SqlAdminPassword);
+			IRelativityVersionHelper relativityVersionHelper = new RelativityVersionHelper(connectionHelper);
+
+			// Confirm Installer and Instance Relativity Version Are Equal
+			relativityVersionHelper.ConfirmInstallerAndInstanceRelativityVersionAreEqual(InstallerRelativityVersion);
+		}
+
+		private void ValidateInputArguments()
+		{
+			if (string.IsNullOrWhiteSpace(RelativityInstanceName))
+			{
+				throw new ArgumentNullException(nameof(RelativityInstanceName), $"{nameof(RelativityInstanceName)} cannot be NULL or Empty.");
+			}
+
+			if (string.IsNullOrWhiteSpace(RelativityAdminUserName))
+			{
+				throw new ArgumentNullException(nameof(RelativityAdminUserName), $"{nameof(RelativityAdminUserName)} cannot be NULL or Empty.");
+			}
+
+			if (string.IsNullOrWhiteSpace(RelativityAdminPassword))
+			{
+				throw new ArgumentNullException(nameof(RelativityAdminPassword), $"{nameof(RelativityAdminPassword)} cannot be NULL or Empty.");
+			}
+
+			if (string.IsNullOrWhiteSpace(SqlAdminUserName))
+			{
+				throw new ArgumentNullException(nameof(SqlAdminUserName), $"{nameof(SqlAdminUserName)} cannot be NULL or Empty.");
+			}
+
+			if (string.IsNullOrWhiteSpace(SqlAdminPassword))
+			{
+				throw new ArgumentNullException(nameof(SqlAdminPassword), $"{nameof(SqlAdminPassword)} cannot be NULL or Empty.");
+			}
+
+			if (string.IsNullOrWhiteSpace(InstallerRelativityVersion))
```

Comment:
Can we also validate that the InstallerRelativityVersion is a valid Version number?